### PR TITLE
Add example product SBOM

### PR DESCRIPTION
**Pull request by twaugh**

### CODE REVIEW

File: sbom/examples/product/create.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    # Collect component purls
+    packages = [
+        {
+            "SPDXID": "SPDXRef-product",
+            "name": name,
+            "versionInfo": versionInfo,
+            "supplier": "Organization: Red Hat",
+            "downloadLocation": "NOASSERTION",
+            "packageFileName": "",
+            "licenseConcluded": "",
```
Comment (twaugh) [inline sbom/examples/product/create.py:16]: Not filled in yet, but these would have values.

File: sbom/examples/product/create.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    for n, described_package in enumerate(described_packages):
+        spdxref = f"SPDXRef-component-{n}"
+        described_package["SPDXID"] = spdxref
+        packages.append(described_package)
+
+        relationships.append(
+            {
+                "spdxElementId": "SPDXRef-product",
+                "relationshipType": "CONTAINS",
```
Comment (twaugh) [inline sbom/examples/product/create.py:60]:
```suggestion
                "relationshipType": "PACKAGE_OF",
```

File: sbom/examples/product/create.py (hunk 1 above)
Comment (mprpic) [reply sbom/examples/product/create.py]: Do products themselves have different licenses that aren't just a compilation of all of the licenses of the individual components that are included in them?

Comment (twaugh) [reply sbom/examples/product/create.py]: Oh, I don't think so. So maybe we'd leave out `licenseConcluded`.

**Comment by mprpic** [comment]
Closing this PR in favor of #5.

**Pull request closed without merging**